Simplified/Settings/NYPLSettings.swift

```diff
@@ -12,7 +12,11 @@ import Foundation
   var accountMainFeedURL: URL? { get set }
 }
 
-@objcMembers class NYPLSettings: NSObject, NYPLFeedURLProvider {
+@objc protocol NYPLAgeCheckChoiceStorage {
```

Comment:
I would move this to the NYPLAgeCheck class, because that's where it's actually used. That way the AgeCheck api is all defined there.